tests/unit/connect/http_client.cpp

```diff
@@ -1,4 +1,4 @@
-#include <httpc.hpp>
+#include "http/httpc.hpp"
```

Comment:
Path include.

Also, this whole test file seems like it's also a good candidate to move, because it's testing the moved file.